### DISCUSSION TITLE
Add option to Highlight Broadcasters messages

### DISCUSSION THREAD

**Discussion by elmarceloc** [Chatterino/chatterino2#3348] [Ideas] [answered]
If its possible to add an option to highlight messages from others streamers (broadcasters) ?

**Comment by Felanbird** [comment, marked as answer]
This feature already exists in ⚙️ -> Highlights -> Badges
![image](https://user-images.githubusercontent.com/41973452/141193702-9eb926e5-54d7-4e3c-b4dc-405d6023d70e.png)
